deploy/openshift/templates/jaeger-all-in-one-template.yml

```diff
@@ -0,0 +1,190 @@
+#
+# Copyright 2017-2018 The Jaeger Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+
+parameters:
```

Comment:
It doesn't change much but in other our templates we specify parameters in the end. Or we can't modify it because of license?

Reply:
I don't want to modify it since its original example and  I don't want to own this modification and want to keep it as is.